library/blivet.py

```diff
@@ -411,7 +412,7 @@ def _update_from_device(self, param_name):
     def _apply_defaults(self):
         global volume_defaults
         for name, default in volume_defaults.items():
-            if name in self._volume:
+            if name in self._volume and self._volume[name] not in [None, list(), dict()]:
```

Comment:
what does it mean for `self._volume[name]` to be `None`?  That is - that appears to be a valid configuration?  Does it mean "use the defaults"?
I think you want to use `isinstance(self._volume[name], list, dict, None)` instead of what you have here

Reply:
If `self._volume[name]` is not None, a dict, or a list, should that be an error for the user?